testit-adapter-playwright/src/reporter.ts

```diff
@@ -62,6 +62,9 @@ class TmsReporter implements Reporter {
     if (step.category !== "test.step") {
       return;
     }
+    if (step.parent !== undefined) {
```

Comment:
Лучше сразу превентивно на будущее и на null проверять, если оно не предполагается как наличие. Универсальный вариант:
```suggestion
    if (!step.parent) {
```